testsuite/forge-test-runner-template.yaml

```diff
@@ -58,8 +58,8 @@ spec:
           value: "1"
         - name: KUBECONFIG
           value: {KUBECONFIG}
-      # - name: RUST_LOG
-      #   value: debug
+        - name: RUST_LOG
```

Comment:
remember to remove this

Reply:
we actually need this. it's info logs so should be ok.